Taskfile.yml

```diff
@@ -179,23 +179,30 @@ tasks:
   test-unit-short:
     desc: "Run short unit tests (with caching)"
     cmds:
+      - go test -c -o /dev/null ./...
       - go test -short -timeout={{.TEST_TIMEOUT}} {{.RACE_FLAG}} -tags={{.BUILD_TAGS}} -shuffle=on -coverprofile=cover.txt ./...
       - bin/task{{exeExt}} -d tools tools-test-short
 
   test-unit:
     desc: "Run all unit tests"
     cmds:
+      - go test -c -o /dev/null ./...
       - go test -count=1 -timeout={{.TEST_TIMEOUT}} {{.RACE_FLAG}} -tags={{.BUILD_TAGS}} -shuffle=on -coverprofile=cover.txt ./...
       - go test -count=1 -timeout={{.TEST_TIMEOUT}} {{.RACE_FLAG}} -tags={{.BUILD_TAGS}} -shuffle=on -bench=. -benchtime=1x ./...
       - bin/task{{exeExt}} -d tools tools-test
 
+  test-integration-build:
+    run: once
+    dir: integration
+    cmd: go test -c -o /dev/null ./...
+
   test-integration:
     desc: "Run integration tests in parallel"
     deps: [test-integration-postgresql, test-integration-mongodb]
 
   test-integration-postgresql:
     desc: "Run integration tests for PostgreSQL/DocumentDB"
-    deps: [gen-version]
+    deps: [gen-version, test-integration-pre]
```

Comment:
CI says it can't find `test-integration-pre`, did you mean to use `test-integration-build` :thinking: ?